tket/src/Utils/include/Utils/UnitID.hpp

```diff
@@ -35,7 +35,7 @@
 namespace tket {
 
 /** Type of information held */
-enum class UnitType { Qubit, Bit };
+enum class UnitType { Qubit, Bit, WasmWireUID };
```

Comment:
Not sure `WasmWireUID` is quite right -- `Qubit` and `Bit` refer to linear types, so ideally this should too: how about `WasmState`?

Reply:
I don't see this change...

Reply:
Forgot to push, should be visible now in @cqc-melf
[rename WasmWireUID to WasmState](https://github.com/CQCL/tket/pull/743/commits/b5ba9ec2536bf6828f02868dad3299649605e5cf)
[b5ba9ec](https://github.com/CQCL/tket/pull/743/commits/b5ba9ec2536bf6828f02868dad3299649605e5cf)